src/components/ResourceList/tests/ResourceList.test.tsx

```diff
@@ -369,6 +369,20 @@ describe('<ResourceList />', () => {
         false,
       );
     });
+
+    it('renders on noninitial load when items are provided', () => {
+      const resourceList = mountWithAppProvider(
+        <ResourceList
+          bulkActions={bulkActions}
+          items={[]}
+          renderItem={renderItem}
+        />,
+      );
+      expect(findByTestID(resourceList, 'ResourceList-Header')).toHaveLength(0);
```

Comment:
```suggestion
const resourceListHeader = findByTestID(resourceList, 'ResourceList-Header');
       expect(resourceListHeader).toHaveLength(0);
```

Reply:
Cool I wasn't aware you could suggest for multiple lines!!

Reply:
They need to stay as separate finds since the components changed between the two searches.

Reply:
I didn't realize `findByTestID` got different instances of the same element.

Reply:
`findByTestID` will find the `ReactWrapper`. However it holds the reference to the `ReactWrapper`. From my understanding `enzyme` can create a new instance. Which is what Im assuming happens when `setProps` and `update` are called.